example/lib/main.dart

```diff
@@ -107,7 +106,7 @@ class NavigationExample extends StatelessWidget {
               );
             },
             itemSeparatorBuilder: (context, index) {
-              return Divider();
+              return Divider(height:1);
```

Comment:
is there a reason this divider gets height 1 instead of using DividerTheme like all other dividers?